.jenkins/jenkins_pr_win.bat

```diff
@@ -0,0 +1,51 @@
+REM CUDNN PATH
```

Comment:
This seem much duplicate code. Refactoring would be useful, but not mandatory for this PR.